src/display/transport_stream.js

```diff
@@ -13,6 +13,11 @@
  * limitations under the License.
  */
 
+/** @typedef {import('../interfaces').IPDFStream} IPDFStream */
+/** @typedef {import('../interfaces').IPDFStreamReader} IPDFStreamReader */
+// eslint-disable-next-line max-len
+/** @typedef {import('../interfaces').IPDFStreamRangeReader} IPDFStreamRangeReader */
```

Comment:
Please use double quotes, as everywhere else.
```suggestion
/** @typedef {import("../interfaces").IPDFStream} IPDFStream */
/** @typedef {import("../interfaces").IPDFStreamReader} IPDFStreamReader */
// eslint-disable-next-line max-len
/** @typedef {import("../interfaces").IPDFStreamRangeReader} IPDFStreamRangeReader */
```